build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -26,7 +30,5 @@
     <file src="$configuration$\Output\$id$.xml" target="lib\net46\$id$.xml" />
     <file src="$configuration$-NetCore\Output\$id$.dll" target="lib\netstandard1.3\$id$.dll" />
     <file src="$configuration$-NetCore\Output\$id$.xml" target="lib\netstandard1.3\$id$.xml" />
-    <file src="$configuration$\Output\ref\$id$.dll" target="ref\net46\$id$.dll" />
-    <file src="$configuration$-NetCore\Output\ref\$id$.dll" target="ref\netstandard1.3\$id$.dll" />
```

Comment:
Don't we still want a ref assembly for 1.3 so that a consumer can be confident that they're targeting the intersection of the API surfaces?